### PR TITLE
Update prettier config and fix

### PR DESCRIPTION
**Pull request by jkronz**
Update the prettierrc.json to more closely match callrail standards and
then run `prettier --write` to fix all the errors.

While prototyping this github action I was committing directly to the main branch without review, but this PR happens to touch every source file in the project so I think it's a good place to get first critical eyes on my typescript source. I know that the diff is mostly semicolons, but if any reviewers want to take a deeper look at the source please feel free.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
       "project": "./tsconfig.json"
     },
     "rules": {
+      "no-console": "off",
```
Comment (jkronz) [inline .eslintrc.json:11]: Logging to console is visible in github actions UI so I find console.log useful and therefore allowed.

File: src/main.ts [hunk 2]
```diff
@@ -12,22 +12,22 @@ function getActionInput(): Input {
...
     packageName: getInput('package-name', { required: true }),
     token: getInput('token', { required: true }),
-    dryRun: getInput('dry-run') ? getInput('dry-run') == 'true' : false,
+    dryRun: getInput('dry-run') ? getInput('dry-run') === 'true' : false
   });
 }
 
 function run(): Observable<boolean> {
   try {
     return deleteVersions(getActionInput()).pipe(
```
Comment (jkronz) [inline src/main.ts:21]: This is the entrypoint to the action. `getInput` functions above pull in the input values defined by the individual project's workflow yaml ([see parameter doc here](https://github.com/callrail/package-cleanup/blob/main/action.yml#L4-L39)). 

`deleteVersions()` is in delete.ts above, which fetches versions from the Github GraphQL API, checks if they match the input `branch-name` and if so sends a mutation request back to the API to delete the version.

File: src/version/delete-version.ts [hunk 3]
```diff
@@ -58,13 +58,13 @@ export function deletePackageVersions(
     deletePackageVersion(id, token).pipe(
```
Comment (jkronz) [inline src/version/delete-version.ts:58]: Loop over the found version IDs and pass them to the API to be deleted. This file is mostly taken from the official github action to [delete oldest packages](https://github.com/marketplace/actions/delete-package-versions), with the exception of me adding a `dryRun` mode that just prints the versions to be deleted and exists without deleting.

File: src/version/get-versions.ts [hunk 4]
```diff
@@ -79,25 +79,20 @@ export function getOldestVersions(
...
+        );
       }
 
       const versions = result.repository.packages.edges[0].node.versions.edges;
 
       return versions
-        .filter(value => value.node.version.indexOf(branchName) >= 0)
-        .map(value => ({id: value.node.id, version: value.node.version}))
-        .reverse()
+        .filter(value => value.node.version.includes(branchName))
```
Comment (jkronz) [inline src/version/get-versions.ts:93]: This line is what I added to filter the returned versions from the API to match branchName that we get from the input.

**Review by jkronz — COMMENTED** [review]
Leaving comments inline

**Review by milesstanfield — APPROVED** [review]
🍭

**Review by getabetterpic — APPROVED** [review]
Nice! 🛠️

**Pull request merged**